scripts/combine/setupCombineWMass.py

```diff
@@ -73,6 +73,8 @@ def main(args):
     tag = name+"_"+args.fitvar.replace("-","_")
     if args.doStatOnly:
         tag += "_statOnly"
+    if args.append:
```

Comment:
In many scripts we have an option --postfix that does a similar thing on the output file name, should we name this option accordingly?

Reply:
I used append because -p was taken and because it impacts the folder not the file name. But I don't feel that strongly